### PR TITLE
changelog: images update part2

### PR DESCRIPTION
**Pull request by davlgd**
## Describe your PR

I've added an entry to the new images update and changed default version supported by .Net in documentation at the same time.

### CODE REVIEW

**Review by cnivolle — DISMISSED** [review]

File: content/changelog/2024-02-27-images-update-part2.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  - update
+authors:
+  - name: David Legrand
+    link: https://github.com/davlgd
+    image: https://github.com/davlgd.png?size=40
+description: The best is yet to come
+excludeSearch: true
+---
+
+Earlier this month, we published [a first set of updates](/changelog/2024-02-02-images-update/) for Docker, Erlang, Go, Haskell, Ruby and Rust. Some days ago, we finished this work and all our images are now up-to-date. This process occurred with no impact for our users.
```
Comment (juliamrch) [inline content/changelog/2024-02-27-images-update-part2.md:15]:
```suggestion
Earlier this month, Clever Cloud published [a first set of updates](/changelog/2024-02-02-images-update/) for Docker, Erlang, Go, Haskell, Ruby and Rust. Some days ago, we finished this work and all runtime images are now up-to-date. This process occurred with no impact for our users. 
```

File: static/partials/language-specific-deploy/dotnet.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 ### .NET version
 
-The default version used on Clever Cloud is `6.0`. You can change it to `5.0` by setting the `CC_DOTNET_VERSION` environment variable to `5.0`. No support will be provided for end-of-life versions.
+The default version used on Clever Cloud is `8.0`. You can change it by setting the `CC_DOTNET_VERSION` environment variable to `6.0`. No support will be provided for non-LTS and older versions.
```
Comment (juliamrch) [inline static/partials/language-specific-deploy/dotnet.md:6]:
```suggestion
The default version used on Clever Cloud is `8.0`. You can change it by setting the `CC_DOTNET_VERSION` environment variable to `6.0`. Clever Cloud doesn't provide support for non-LTS and older versions.
```

File: static/partials/language-specific-deploy/dotnet.md [hunk 3]
```diff
@@ -17,10 +17,10 @@ Let's take an example with the [simple-feed-reader project](https://github.com/d
 
 First, you need to add the `APP_FOLDER=SimpleFeedReader` environment variable to define the application folder inside the Git repository.
 
-During deployment, the `SimpleFeedReader.csproj` file and the target framework `net5.0` are automatically detected. Then, the .NET project is published:
+During deployment, the `SimpleFeedReader.csproj` file and the target framework `net8.0` are automatically detected. Then, the .NET project is published:
```
Comment (juliamrch) [inline static/partials/language-specific-deploy/dotnet.md:20]:
```suggestion
During deployment, the runtime automatically detects the `SimpleFeedReader.csproj` file and the target framework `net8.0`. Then, it publishes the .NET project:
```

**Review by juliamrch — CHANGES_REQUESTED** [review]
Nice update, added some suggestions to clarify what ad who performs the action (runtime, Clever Cloud). Feel free to suggest other versions.

**Review by juliamrch — DISMISSED** [review]
Big fan of the `stay tuned` with the RSS link. Let's merge.

**Comment by davlgd** [comment]
> Big fan of the `stay tuned` with the RSS link. Let's merge.

We need to wait @s0dyy review before merging

**Comment by juliamrch** [comment]
waiting for @s0dyy to have a look before merging.

- [Changelog entry](http://doc-review-pr-210.cleverapps.io/changelog/2024-02-27-images-update-part2/)
-  [.NET variables](http://doc-review-pr-210.cleverapps.io/doc/reference/reference-environment-variables/#net)
- [.Net page update](http://doc-review-pr-210.cleverapps.io/doc/applications/dotnet/#net-version)

File: content/changelog/2024-02-27-images-update-part2.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    image: https://github.com/davlgd.png?size=40
+description: The best is yet to come
+excludeSearch: true
+---
+
+Earlier this month, we published [a first set of updates](/changelog/2024-02-02-images-update/) for Docker, Erlang, Go, Haskell, Ruby and Rust. Some days ago, we finished this work and all images are now up-to-date. This process occurred with no impact for our users. 
+
+As mentioned in a previous post, this enables [the new `healtcheck` feature](/changelog/2024-02-26-healthcheck-for-everyone/) available for all applications. New images include security patches, Linux kernel 6.7.1, OpenSSL 3.2.1 and Node.js 20.11.0 by default. For the latter, you can change it in the updated images via the `CC_NODE_VERSION` [environment variable](/doc/reference/reference-environment-variables/#commons-to-all-applications).
+
+.Net `8.0` version is now available and enabled by default for new applications. This value can be modified via [the `CC_DOTNET_VERSION` environment variable](/doc/reference/reference-environment-variables/#net). PHP 8.3 is next to come, we're currently working on it. The Elastic / APM agent has been updated for Java users and is now compatible with Java 21.
```
Comment (s0dyy) [inline content/changelog/2024-02-27-images-update-part2.md:19]: Same here, not very readable and everything is mixed up (dotnet, php, apm...).

I prefer something simpler, like :

Dotnet:

- Version 8 support
- CC_DOTNET_VERSION variable now created by default for each new Dotnet application

Java:
- Version bump to 1.47.1 (With Java 21 support)

File: content/changelog/2024-02-27-images-update-part2.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+As mentioned in a previous post, this enables [the new `healtcheck` feature](/changelog/2024-02-26-healthcheck-for-everyone/) available for all applications. New images include security patches, Linux kernel 6.7.1, OpenSSL 3.2.1 and Node.js 20.11.0 by default. For the latter, you can change it in the updated images via the `CC_NODE_VERSION` [environment variable](/doc/reference/reference-environment-variables/#commons-to-all-applications).
+
+.Net `8.0` version is now available and enabled by default for new applications. This value can be modified via [the `CC_DOTNET_VERSION` environment variable](/doc/reference/reference-environment-variables/#net). PHP 8.3 is next to come, we're currently working on it. The Elastic / APM agent has been updated for Java users and is now compatible with Java 21. 
+
+Next step of our big images overhaul is to bring our new strategy to build them into production. We're currently testing it, and preparing enhancements for multiples runtimes. Your applications will automatically benefit from it when available. [Stay tuned](/changelog/index.xml).
```
Comment (s0dyy) [inline content/changelog/2024-02-27-images-update-part2.md:21]: No need to talk about it

Comment (juliamrch) [reply content/changelog/2024-02-27-images-update-part2.md:21]: Thanks for your review @s0dyy

**Comment by juliamrch** [comment]
Hi @davlgd the `static/partials/language-specific-deploy/dotnet.md` file doesn't exist anymore in the codebase, I think you can resolve conflicts by simply adding the content you modified to `content/doc/applications/dotnet/_index.md`.

**Comment by davlgd** [comment]
LGTM

**Review by juliamrch — APPROVED** [review]
Conflicts seem resolved. Nice update. Good to go imo

**Pull request merged**